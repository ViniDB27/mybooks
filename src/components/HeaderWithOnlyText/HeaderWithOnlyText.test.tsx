import render from "react-test-renderer";

import { HeaderWithOnlyText } from "./index";

describe("<HeaderWithOnlyText />", () => {
  const tree = render.create(
    <HeaderWithOnlyText text="Mock" />
  ).toJSON();
  
  
  it("renders correctly", () => {
    expect(tree).toMatchSnapshot();
  });
});
