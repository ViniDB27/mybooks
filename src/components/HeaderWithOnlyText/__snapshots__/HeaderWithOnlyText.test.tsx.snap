// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderWithOnlyText /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0Af",
      "height": 80,
      "justifyContent": "center",
      "position": "relative",
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 26,
        "fontWeight": "bold",
      }
    }
  >
    Mock
  </Text>
</View>
`;
